conf/systemd.service

```
@@ -1,13 +1,13 @@
 [Unit]
-Description=13ft launcher service
+Description=13ft: launcher service
 After=network.target
 
 [Service]
 Type=simple
 User=__APP__
 Group=__APP__
-WorkingDirectory=__FINALPATH__/
-ExecStart=__FINALPATH__/venv/bin/python3 __FINALPATH__/index.py
+WorkingDirectory=__INSTALL_DIR__/
+ExecStart=__INSTALL_DIR__/venv/bin/python3 __INSTALL_DIR__/index.py
```

```suggestion
ExecStart=__INSTALL_DIR__/venv/bin/python3 __INSTALL_DIR__/app/index.py
```

Dunno how it was working before ...

---

It wasn't...